lib/apiservers/engine/backends/container.go

```diff
@@ -232,16 +241,16 @@ func (c *Container) ContainerExecCreate(name string, config *types.ExecConfig) (
 		}
 
 		switch state {
-		case "STOPPED":
+		case StoppedState:
```

Comment:
you can use a comma separated list in a case statement:
```
switch state {
case CreatedState, StoppedState, SuspendedState:
   return InternalServerError(fmt.Sprintf("Container (%s) is not running", name))
case StartingState:
   return ConflictError(fmt.Sprintf("container (%s) is still starting", id))
case RunningState:
   // noop
default:
   return InternalServerError(fmt.Sprintf("Container (%s) is in an unknown state: %s", id, state))
}

Reply:
Good point, will fix this. I knew I was doing something silly there when I had so many wasted lines.